src/Test/TestCommanderBrenner.java

```diff
@@ -92,22 +92,27 @@ private boolean testCOU()
     return testPassed;
   }
 
+  /**
+   * Verifies that the zone is the only place Brenner gets energy.<p>
+   * The charge bug (combat initiator must be in your zone, not your unit) is not checked here.
+   */
   private boolean testAttackCharge()
   {
     setupTest();
 
     // Add our combatants
-    Unit mech = addUnit(testMap, Brenner, UnitModel.MECH, 1, 1);
+    Unit mech = addUnit(testMap, Brenner, UnitModel.MECH, 1, 1); // 4 tiles from COU, so not in zone
     Unit COU  = addUnit(testMap, Brenner, UnitModel.TRANSPORT, 4, 2);
-    Unit aa   = addUnit(testMap, Brenner, UnitModel.SURFACE_TO_AIR, 2, 2);
+    Unit aa   = addUnit(testMap, Brenner, UnitModel.SURFACE_TO_AIR, 2, 2); // 2 tiles from COU, so in zone
 
     // And our victim
     Unit inf = addUnit(testMap, Patch, UnitModel.TROOP, 1, 2);
 
     Brenner.initTurn(testMap);
     Brenner.army.money = 9001;
     boolean testPassed = true;
-    
+
+    // Note: COUing on this tile is not normally valid, but we're skipping the logic that enforces that.
```

Comment:
Could just swap the attack command so the inf attacks the mech instead to test that case. Shrug.

Reply:
I don't think that comment applies to this line?
Either way, I'm not worried about having a test for the charge bug, especially since I may be putting in an option for it at some point.